src/events/addComment.js

```diff
@@ -0,0 +1,8 @@
+export default () => {
+  // const submitBtn = document.querySelector('.submitBtn')
+  // console.log(submitBtn);
+  // submitBtn.addEventListener('click', (e) => {
+  //     e.preventDefault()
+  //     alert("hello")
+  // })
```

Comment:
- [ ] Please remove commented codes from your project. This would make your code cleaner.